frontend/lib/src/components/elements/DeckGlJsonChart/useDeckGl.tsx

```diff
@@ -154,15 +156,12 @@ function updateWidgetMgrState(
 
 export const useDeckGl = (props: UseDeckGlProps): UseDeckGlShape => {
   const {
-    element,
-    fragmentId,
     height: propsHeight,
-    isFullScreen: propsIsFullScreen,
-    isLightTheme,
-    theme,
-    widgetMgr,
     width: propsWidth,
-  } = props
+    expanded: propsIsFullScreen,
```

Comment:
super nit: these are coming from the context and not from the passed `props` anymore, so perhaps `context*` might be a more accurate prefix